en/flying/fixed_wing_takeoff.md

```diff
@@ -0,0 +1,56 @@
+# Fixed Wing Takeoff
+
+PX4 supports fixed-wing takeoff in [missions](#mission) and using the [Takeoff](#takeoff) flight mode.
+Vehicles can be *catapult/hand-launched* or use the runway takeoff mode. <!-- runway support in missions? -->
```

Comment:
Do we support runway mode in mission mode? I have assumed not.

Reply:
I'm not sure...  @ryanjAA do you know?

Reply:
@ryanjAA @RomanBapst Can you confirm?

Reply:
@hamishwillee Yes, we do

Reply:
Awesome. As we support catapult/runway takeoff in both takeoff mode and missions, this text is correct. Setting resolved.